### PR TITLE
feat : added note home page

### PR DESCRIPTION
**Pull request by rohit8686**
Features : 

- Add Notes (title, body, label, color)
- Edit and update Notes 
- Pinned and unpinned notes displayed separately
- Responsive note home page

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=ROHIT
```
Comment (OmKakatkar) [inline .env:1]: For security reasons, `env` file is generally not commited to git. You can easily add an environment variable on your hosting site for this.

**Review by OmKakatkar — APPROVED** [review]
Over all looks good
Like:
* Modular code with separate folders for components
* Meaningful variable and class names
*  Separate Providers for distinct tasks

Suggestion:
* Have a look at the comment below

**Review by RahulSingh9131 — APPROVED** [review]
everything is perfectly written, liked the idea of breaking things into smaller usable components. css is clean too.

**Review by mdarif6 — APPROVED** [review]
- Liked naming convention.
- Liked theme provider.
- Folder structure is also good.

**Pull request merged**